Example/Example/Shared/Configuration.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Configuration.swift
+//  Example
+//
+//  Created by Ryan Davis on 4/8/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+import Foundation
+import os.log
+
+private extension TimeInterval {
+  static var oneDay: Self { 60 * 60 * 24 }
+}
+
+private enum UserDefaultsKey {
+  static var configuration: String { #function }
+  static var lastFetchDate: String { #function }
+}
+
+func getCachedConfiguration(
```

Comment:
🤔 this function name is a little misleading as it won't always get the 'cached' configuration if it doesn't already have a cache

Reply:
Yeah it could be better. Got any ideas?

Reply:
If you want to it could be time to build some lightweight encapsulation. E.g. the app must go through a layer that may or may not cache things